main.js

```diff
@@ -2,3 +2,43 @@
 // Go back to the lesson if you feel lost
 // Read the docs well, they usually have pretty well documented examples
 // Consider making your fetch work with a button
+
+
+
+const img = document.createElement('img');
+const button = document.querySelector('#random');
+const apiUrl = `https://rickandmortyapi.com/api/character/`;
+// random.innerText = "Random";
+const RickAndMortyAPI = () => {
+    for (i = 1; i <= 5; i++) {
+        axios({
+
+            url: apiUrl + [i],
+
+            method: 'get',
+        }
+
+        )
+            .then(res => {
+                console.log(res)
+                // const arr = res.data.results;
```

Comment:
You can add a new image with each loop so you can display all photo together 
```suggestion
      const img = document.createElement('img');
```
I think you also forget to display the name with the photo